addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2913,6 +2913,11 @@ export class Wysiwyg extends Component {
     }
     _attachHistoryIds(editable = this.odooEditor.editable) {
         if (this.options.collaborative) {
+            // clean existig 'data-last-history-steps' attributes
```

Comment:
Why though ?

Reply:
For now we have some 'data-last-history-steps' that can appear all over the document. and they interfere with the diffing process ( create false change diff when not necessary).

They are created when you have create a new line `[ENTER]` in the first block of the document. this block contain 'data-last-history-steps' attribute and will be copied to the new block element.

I've confirm with NBY that we only need the 'data-last-history-steps' in the first block element and that the attribute is regenerated every time it is needed for saving anyway. 
So I took the liberty to clean all the rogue 'data-last-history-steps' in the document.